src/nodekit.ts

```diff
@@ -91,25 +91,33 @@ export class NodeKit {
             utils: this.utils,
         });
 
-        this.addShutdownHandler(() => {
-            return new Promise((resolve) => {
-                this.tracer.close?.(() => resolve());
```

Comment:
Was this optional chaining function call intentional? If it was, is it jaeger client typings issue or something else?

Reply:
Now ts doesn't see any problem here. I will remove this.